server/src/main/resources/config/nodeSauceConfig.json

```diff
@@ -2,16 +2,8 @@
     "class": "org.openqa.grid.common.RegistrationRequest",
     "capabilities": [{
         "platform": "ANY",
-        "browserName": "firefox",
-        "maxInstances": 20
-    }, {
-        "platform": "ANY",
-        "browserName": "chrome",
-        "maxInstances": 20
-    }, {
-        "platform": "ANY",
-        "browserName": "internet explorer",
-        "maxInstances": 20
+        "browserName": "saucelabs",
```

Comment:
This is the selenium convention.. some mobile drivers refer to mobile apps as "browsers" too.

When someone uses our sauce labs proxy, they can pass any "browserName" that sauce labs supports (https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/) -- sauce labs will perform the match and honor it.

For example:
- selion `@WebTest` example
```java
@WebTest(browser = "microsoftedge", additionalCapabilities = { "platform:Windows 10", "version:13.10586" })
```

- raw node.js example
```javascript
caps = {browserName: 'MicrosoftEdge'};
caps['platform'] = 'Windows 10';
caps['version'] = '13.10586';
```

What's important here is the slot type which is `saucelabs` ... DesiredCapability `browserName="saucelabs"` would never be passed by a client and if it was sauce labs would fail to match against the capability.

Prior to the change a client could ask for "browserName=safari", saucelabs would match and honor it, yet the sauce proxy would map it to a slot with the firefox, chrome, or ie icon.. That's not accurate in the slightest.